tox.ini

```diff
@@ -14,6 +14,7 @@ envdir =
     py3{4,5,6,7,8,}{-unit,-functional,-style}: {toxworkdir}/py3
     docs: {toxworkdir}/docs
 deps =
+    NEURON
```

Comment:
I think it's better to use nightly here, to catch bugs early

Reply:
Ah good point.